README.md

```diff
@@ -113,6 +113,7 @@ ignore_missing=true
 | ignore_missing | Ignore missing commits previously used in the release | default: false | no | - |
 | bin_dir | The folder where the plugin downloads the sentry-cli binary | .dart_tool/pub/bin/sentry_dart_plugin (string) | no | - |
 | bin_path | Path to the sentry-cli binary to use instead of downloading. Make sure to use the correct version. | null (string) | no | - |
+| sentry_cli_cdn_url | Alternative place to download sentry-cli | default: https://downloads.sentry-cdn.com/sentry-cli | no | SENTRYCLI_CDNURL |
```

Comment:
I'd call the env var `SENTRY_CLI_CDN_URL` as it is easier to read and also mirrors the format of `sentry_cli_cdn_url`

Reply:
The env var name `SENTRYCLI_CDNURL` comes from the official doc https://docs.sentry.io/cli/installation/#downloading-from-a-custom-source, I've used it in JS project, so I think different sentry SDK can share the same env name.

Should we support both `SENTRY_CLI_CDN_URL` and `SENTRYCLI_CDNURL`, and make `SENTRYCLI_CDNURL` as fallback, or just `SENTRY_CLI_CDN_URL`?